### PR TITLE
Fixed spaces absence between blocks on ship and billing steps

### PR DESCRIPTION
**Pull request by Mazenatef1**
**Related issue(s):**
* Fixes https://github.com/scandipwa/scandipwa/issues/3551

**Problem:**
Space is absent between promotion block and summary block on shipping and billing steps

**In this PR:**
added a margin to CmsBlock-Wrapper by 15px

### CODE REVIEW

File: packages/scandipwa/src/component/CmsBlock/CmsBlock.style.scss [hunk 1]
```diff
@@ -10,6 +10,10 @@
  */
 
 .CmsBlock {
+    &-Wrapper {
+        margin: 15px;
+    }
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/CmsBlock/CmsBlock.style.scss:15]: Won't this break for other cms blocks on website?

**Review by AleksandrsKondratjevs — CHANGES_REQUESTED** [review]

**Review by AleksandrsKondratjevs — APPROVED** [review]

File: packages/scandipwa/src/component/CmsBlock/CmsBlock.style.scss [hunk 2]
```diff
@@ -10,6 +10,7 @@
  */
 
 .CmsBlock {
+
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/CmsBlock/CmsBlock.style.scss:13]: redundant white space

**Review by AleksandrsKondratjevs — CHANGES_REQUESTED** [review]
See comments

File: packages/scandipwa/src/component/CheckoutOrderSummary/CheckoutOrderSummary.style.scss [hunk 3]
```diff
@@ -22,6 +22,7 @@
         padding-block-start: 24px;
         padding-block-end: 0;
         padding-inline: 16px;
+        margin-bottom: 15px;
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/CheckoutOrderSummary/CheckoutOrderSummary.style.scss:25]: Use logic property.

**Review by AleksandrsKondratjevs — CHANGES_REQUESTED** [review]
See comments.

File: packages/scandipwa/src/component/CheckoutOrderSummary/CheckoutOrderSummary.style.scss [hunk 4]
```diff
@@ -19,9 +19,9 @@
     @include desktop {
         border: 1px solid var(--secondary-base-color);
         border-radius: 2px;
-        padding-block-start: 24px;
-        padding-block-end: 0;
+        padding-block: 24px 0;
         padding-inline: 16px;
+        margin: 0 0 15px 0;
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/CheckoutOrderSummary/CheckoutOrderSummary.style.scss:24]: Instead can you use only margin-block-end: 15 px? Or it is required to be all other nulls?

Comment (chewdisout) [reply packages/scandipwa/src/component/CheckoutOrderSummary/CheckoutOrderSummary.style.scss:24]: Basically as i understand it should be all other nulls (if i get it right). But i don't think it is required, so will change

**Review by AleksandrsKondratjevs — APPROVED** [review]

**Pull request merged**